examples/nginx-access-logs-streaming/README.md

```diff
@@ -0,0 +1,217 @@
+# Analyzing Nginx Access Logs using Timeplus Proton 
+In a [recent article](https://www.timeplus.com/post/log-stream-analysis), we walked through three different ways of using Timeplus Proton to ingest and analyse a log file in real-time.
+
+I’ll explore a similar theme in the first part of this article: ad-hoc analysis of web traffic, in real-time, using Timeplus Proton. I will use a Node.js blog that is behind an Nginx web server for the analysis. 
+
+In the second part of this article, I will analyze past traffic numbers for the blog. I'll compare historical traffic numbers obtained using Timeplus Proton with traffic numbers reported by [Umami](https://umami.is). (Umami is an open source, privacy-focused alternative to Google Analytics.) 
```

Comment:
"traffic numbers" is a bit unconventional

Reply:
Changed to "traffic stats".